src/components/nurseCardDisplay.tsx

```diff
@@ -15,9 +15,9 @@ interface IFormInput {
 }
 
 export function NurseCardDisplay(
-  { staffData }: { staffData: IFormInput[] }) {
+  { staffData, nurseId }: { staffData: IFormInput[], nurseId: string }) {
```

Comment:
Looks like you passed the Nurse ID from another component to this one via props.